test/e2e/transformer/test_raw_transformer.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import requests
+import time
+import logging
+from kubernetes import client
+
+from kfserving import KFServingClient
+from kfserving import constants
+from kfserving import V1beta1PredictorSpec
+from kfserving import V1beta1TransformerSpec
+from kfserving import V1beta1TorchServeSpec
+from kfserving import V1beta1InferenceServiceSpec
+from kfserving import V1beta1InferenceService
+from kubernetes.client import V1ResourceRequirements
+from kubernetes.client import V1Container
+from kubernetes.client import V1EnvVar
+from ..common.utils import get_cluster_ip
+from ..common.utils import KFSERVING_TEST_NAMESPACE
+logging.basicConfig(level=logging.INFO)
+KFServing = KFServingClient(
+    config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+
+
+def test_transformer():
+    service_name = 'raw'
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        pytorch=V1beta1TorchServeSpec(
+            storage_uri='gs://kfserving-examples/models/torchserve/image_classifier',
+            resources=V1ResourceRequirements(
+                requests={'cpu': '1', 'memory': '2Gi'},
```

Comment:
Can we set lower cpu request  like 100m as we have limited resource to run all the e2e tests

Reply:
sure.